src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -183,6 +183,11 @@ public function configureOptions(OptionsResolver $resolver)
             'allow_file_upload' => false,
             'help' => null,
             'help_attr' => array(),
+            'invalid_message' => function (Options $options, $previousValue) {
+                return ($options['legacy_error_messages'] ?? true) ?
+                    'The form is invalid.' :
+                    $previousValue;
+            },
```

Comment:
You must define the `invalid_message_parameters` option here now to be able to use it in the core extension. It also means we're missing a test case for this support.